cub/cub/agent/single_pass_scan_operators.cuh

```diff
@@ -504,6 +558,8 @@ struct ScanTileState;
 template <typename T>
 struct ScanTileState<T, true>
 {
+  using ValueT = T;
```

Comment:
important: other tile states are missing this member

Reply:
Thanks, I made it also a type member of the other tile state template specializations.